public/server/db/bf.json

```
@@ -0,0 +1,4 @@
+
+{
+  "definition": "Boy Friend"
```

In the other file, you added an alternatives field. But you left that out here. Would you like to add that before I merge this in?
```suggestion
  "definition": "Boy Friend",
  "alternatives": "Breakfast"
```